### PR TITLE
New package: elog-3.1.4

### PR DESCRIPTION
**Pull request by the-wrench99**

### CODE REVIEW

File: srcpkgs/elog/template [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# Template file for 'elog'
+pkgname=elog
+_elogrel=1
```
Comment (Vaelatern) [inline srcpkgs/elog/template]: Why is this line here? Maybe should be part of the version? Definitely not up at pkgname,version,revision.

File: srcpkgs/elog/template [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+version=3.1.4
+revision=1
+build_style=gnu-makefile
+make_dirs="/etc/elog 0750 root elog
+/var/lib/elog/logbooks 0750 elog elog"
+system_accounts="elog"
+hostmakedepends=""
+makedepends="libressl-devel"
+depends="libressl"
```
Comment (Vaelatern) [inline srcpkgs/elog/template]: No, no it doesn't. I'll add a travis link if you need convincing.

File: srcpkgs/elog/template [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+short_desc="Logbook system to manage notes through a Web interface"
+maintainer="Christian Poulwey <christian.poulwey@t-online.de>"
+license="GPL-3.0-or-later"
+homepage="https://elog.psi.ch/elog/"
+distfiles="https://elog.psi.ch/elog/download/tar/$pkgname-$version-$_elogrel.tar.gz"
+checksum=4bfa2bef6cfa20aba73c9c6c044a4c83f20cf03e7de3c0dda4a4477f28b7b72e
+
+do_build() {
+	make $makejobs
+}
```
Comment (Vaelatern) [inline srcpkgs/elog/template]: Remove this entire function. It's trash.

File: srcpkgs/elog/template [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+	vmkdir	usr/share/elog
+	
+	vcopy	resources	usr/share/elog
+	vcopy	scripts		usr/share/elog
+	vcopy	themes		usr/share/elog
+
+	sed -i 	'/\[global\]/s/$/\nLogbook dir = \/var\/lib\/elog\/logbooks/' elogd.cfg
+	sed -i '/\[global\]/s/$/\nResource dir = \/usr\/share\/elog/' elogd.cfg
+	vsconf	elogd.cfg
```
[the remark this answers is not in the document]
Comment (Vaelatern) [reply srcpkgs/elog/template]: Because it can't go under /etc directly.
But I want the installation to the dir under /etc to happen here, and `conf_files` to be appropriately set.

File: srcpkgs/elog/files/elogd/run [hunk 5]
```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+exec chpst -u elog:elog /usr/bin/elogd -c /etc/elog/elogd.cfg
```
Comment (Vaelatern) [inline srcpkgs/elog/files/elogd/run]: Actually, why can't you just give it `/etc/elogd.cfg` unless this is where upstream thinks the conf file should go?

File: srcpkgs/elog/template (hunk 2 above)
Comment (Vaelatern) [reply srcpkgs/elog/template]: https://travis-ci.org/void-linux/void-packages/jobs/455704958#L2659-L2662

File: srcpkgs/elog/template (hunk 3 above)
Comment (Vaelatern) [reply srcpkgs/elog/template]: It breaks cross build at least.

File: srcpkgs/elog/template [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+}
+do_install() {
+	vbin	elog
+	vbin	elogd
+	vbin	elconv
+
+	vman	man/elog.1
+
+	vmkdir	usr/share/elog
+
```
Comment (Vaelatern) [inline srcpkgs/elog/template]: xlint warns you of trailing whitespace on this line.

File: srcpkgs/elog/files/elogd/run (hunk 5 above)
Comment (the-wrench99) [reply srcpkgs/elog/files/elogd/run]: upstream recommends a subdirectory in which SSL keys can be placed, too.
source: https://elog.psi.ch/elog/adminguide.html

File: srcpkgs/elog/template (hunk 1 above)
Comment (the-wrench99) [reply srcpkgs/elog/template]: is there a way to better reflect the upstream's naming scheme which looks like $pkgname-$version_$revision?

Comment (Vaelatern) [reply srcpkgs/elog/template]: See my work in `slurm-wlm`
```
pkgname=slurm-wlm
version=18.08.1.1
revision=3
_distver="${version//./-}"
```

Comment (Vaelatern) [reply srcpkgs/elog/template]: There the version scheme is consistent `-` instead of `.` but there are shell features to make these changes.
